CompanyEmployees.Core.Domain/Repositories/IAttendanceRepository.cs

```diff
@@ -0,0 +1,13 @@
+using CompanyEmployees.Core.Domain.Entities;
+
+namespace CompanyEmployees.Core.Domain.Repositories;
+
+public interface IAttendanceRepository
+{
+    Task<IEnumerable<Attendance>> GetAllAttendancesAsync(bool trackChanges, CancellationToken ct = default);
+    Task<Attendance> GetAttendanceAsync(Guid attendanceId, bool trackChanges, CancellationToken ct = default);
+    Task<IEnumerable<Attendance>> GetAttendancesByEmployeeIdAsync(Guid employeeId, bool trackChanges, CancellationToken ct = default);
```

Comment:
Consider renaming `GetAttendanceAsync` to `GetAttendanceByIdAsync` for naming consistency and clarity, matching patterns like `GetEmployeeAsync` and clarifying the parameter purpose.

**Suggested change**
```suggestion
Task<Attendance> GetAttendanceByIdAsync(Guid attendanceId, bool trackChanges, CancellationToken ct = default);
```